From fab0170b7203bec5b704ea90213b0f8d30d34c8b Mon Sep 17 00:00:00 2001
From: Dmytro <dmitry.tkk@gmail.com>
Date: Fri, 31 Jul 2020 18:46:20 +0300
Subject: [PATCH] SKALE-3026 Add py2_warn to hiddenimports

---
 main.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.spec b/main.spec
index cf2c977..e68692d 100644
--- a/main.spec
+++ b/main.spec
@@ -30,7 +30,7 @@ a = Analysis(['./cli/main.py'],
                  (os.path.dirname(wcwidth.__file__), 'wcwidth'),
                  *external_data
                  ],
-             hiddenimports=[],
+             hiddenimports=['pkg_resources.py2_warn'],
              hookspath=[],
              runtime_hooks=[],
              excludes=['tkinter'],
-- 
2.24.3 (Apple Git-128)

